lib/content-services/src/lib/aspect-list/aspect-list.component.spec.ts

```diff
@@ -156,6 +156,12 @@ describe('AspectListComponent', () => {
             fixture.destroy();
         });
 
+        it('should return true when same aspect list selected', () => {
+            expect(component.hasEqualAspect()).toBe(true);
+            component.clear();
+            expect(component.hasEqualAspect()).toBe(false);
```

Comment:
Can we split to 2? 

* Should return true when etc...
* Should return false when etc...